code-blast.js

```diff
@@ -225,33 +225,36 @@ https://twitter.com/JoelBesada/status/670343885655293952
 			canvas.style.position = 'fixed';
 			canvas.style.top = 0;
 			canvas.style.left = 0;
-			canvas.style.zIndex = 1;
+			canvas.style.zIndex = 2147483647;
```

Comment:
Fine, but I'm still not convinced that the largest possible value is required here. :thinking: